.github/PULL_REQUEST_TEMPLATE.MD

```diff
@@ -0,0 +1,15 @@
+Fixes #{Issue Number}
```

Comment:
Let's move this below `description` and add a header `## Related issues and discussions` and change `Fixes #{Issue Number}` to `#{Issue Number}`